### PR TITLE
[Cleanup] Formatting Date Properties For Invoices With Timezone

### PR DESCRIPTION
**Pull request by Civolilah**
@beganovich @turbo124 The PR includes formatting of the `next_send_date` property for Quotes. This property is only displayed on the Quote slider, where I formatted it. We don't have a create column for Quotes as we do for others. Please let me know if I should add a column, and also, we have a `last_sent_date` column - let me know if that one needs to be formatted by timezone as well. Let me know your thoughts.

### CODE REVIEW

File: src/common/hooks/useGetTimezone.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        return {
+          timeZoneId: result.id,
+          timeZone: result.name,
+        };
+      }
+    }
+
+    return {
+      timeZoneId: '1',
+      timeZone: 'America/Tijuana',
```
[the remark this answers is not in the document]
Comment (Civolilah) [reply src/common/hooks/useGetTimezone.ts]: @beganovich We should never encounter this case - it's only returned as a fallback due to other security checks, like the possibility that the `find` function can return a `null` value. However, I set the ID to "32" and the timeZone to "Europe/Lisbon" (which is UTC time), so by doing it your recommendation has been applied (fallback is UTC time). Let me know your thoughts.

File: src/common/hooks/useGetTimezone.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import { useStaticsQuery } from '../queries/statics';
+
+export function useGetTimezone() {
+  const { data: statics } = useStaticsQuery();
+
+  return (timeZoneId: string | undefined) => {
+    if (statics?.timezones && timeZoneId) {
+      const result = statics.timezones.find(
+        (format: any) => format.id === timeZoneId
```
[the remark this answers is not in the document]
Comment (Civolilah) [reply src/common/hooks/useGetTimezone.ts]: @beganovich Ah sorry, missed it. It is fixed.

**Comment by beganovich** [comment]
Ping for merge conflicts :+1:

**Comment by Civolilah** [comment]
> Ping for merge conflicts 👍

@beganovich Resolved!

**Review by beganovich — APPROVED** [review]

**Pull request merged**